source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,250 @@
+===========
+Search Text
+===========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you will learn how to run a **text search** in MongoDB.
+
+You can use a text search to retrieve documents that contain a **term**
+or a **phrase** in a specified field. A term is a sequence of characters
+that excludes whitespace characters. A phrase is a sequence of terms
+that with any number of whitespace characters. 
+
+The following sections show you how to perform the following types of
+text searches:
+
+- Search Text by a Term
+- Search Text by a Phrase
+- Search Text with Terms Excluded
+
+If you want to sort by relevance, see our :ref:`Sort Results
+<sorts-crud-text-search>` page.
```

Comment:
```suggestion
Not sure if a reader will know what relevance is, so maybe omitting that and mentioning that you're sorting results of a text search would be more clear.

```suggestion
If you want to sort your text search results, see the :ref:`Text Search <sorts-crud-text-search>` section of our Sort Results guide.
```